shell/common/shell.cc

```diff
@@ -283,42 +300,76 @@ std::unique_ptr<Shell> Shell::Create(
     const Shell::CreateCallback<PlatformView>& on_create_platform_view,
     const Shell::CreateCallback<Rasterizer>& on_create_rasterizer,
     DartVMRef vm) {
+  auto shell_holder_future = Shell::InitShellEnv(
```

Comment:
Put `TRACE_EVENT0("flutter", "Shell::Create");` as the first line of this function.